ssh-copy-id.sh

```diff
@@ -20,7 +20,7 @@ check_access_and_set_exitcode() {
 ssh_copy_id() {
     info trying to add ssh key to remote authorized_keys file
     info 'you may be prompted for your remote password'
-    ssh $tunnelsite 'mkdir -p .ssh; cat >> .ssh/authorized_keys; chmod -R go-rwx .ssh' < $ssh_key_file.pub
+    ssh $tunnelsite "mkdir -p .ssh; sed -e \"s|__PATH_TO_FALSE__|\`which false\`|\" >> .ssh/authorized_keys; chmod -R go-rwx .ssh" < $ssh_key_file.pub
```

Comment:
I'll change that... Sec... :)

Met vriendelijke groet,

Marius van Witzenburg
"Perfection, is not an act, but a habit."

On Sun, Jun 28, 2015 at 10:54 PM, Janos Gyerik <notifications@github.com>
wrote:

> In ssh-copy-id.sh
> <https://github.com/janosgyerik/autossh-tunnel/pull/3#discussion_r33426799>
> :
>
> > @@ -20,7 +20,7 @@ check_access_and_set_exitcode() {
> >  ssh_copy_id() {
> >      info trying to add ssh key to remote authorized_keys file
> >      info 'you may be prompted for your remote password'
> > -    ssh $tunnelsite 'mkdir -p .ssh; cat >> .ssh/authorized_keys; chmod -R go-rwx .ssh' < $ssh_key_file.pub
> > +    ssh $tunnelsite "mkdir -p .ssh; sed -e \"s|__PATH_TO_FALSE__|\`which false\`|\" >> .ssh/authorized_keys; chmod -R go-rwx .ssh" < $ssh_key_file.pub
>
> Using backticks is deprecated. Please use $(which false) instead
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/janosgyerik/autossh-tunnel/pull/3/files#r33426799>.
>